src/core/core.js

```diff
@@ -140,7 +140,11 @@ export default class Core {
       this.globalStorage.delete(StorageKeys.SEARCH_OFFSET);
     }
 
-    const locationRadiusFilterNode = this.filterRegistry.getFilterNodeByKey(StorageKeys.LOCATION_RADIUS);
+    if (!useFacets) {
+      this.filterRegistry.setFacetFilterNodes([], []);
```

Comment:
For my own edification, why do we need to do this?

Reply:
When you trigger a search with the searchbar, all of the facets are cleared, as opposed to when you trigger a search with facets or any static filters. What was happening was that I would click some facets, apply them, and see them pop up in the applied filters bar. sweet. then I would search with the search bar, and the applied filters bar would pretend those same facets were applied to the search, when in fact they were NOT because useFacets was false (since the search was triggered with the search bar). This was because the facets were still stored in the filter registry, after the search, even though they were not used.